src/lib/attachment-from-file.js

```diff
@@ -5,6 +5,7 @@ module.exports = path => {
   const mime = mimeTypeFor(path);
   return {
     content_type: mime,
+    // eslint-disable-next-line no-buffer-constructor
```

Comment:
This sounds like a legit thing that should be following as opposed to disabled: https://eslint.org/docs/rules/no-buffer-constructor

Reply:
(it is frustrating that ESlint is a combination of style [where do the braces go?], idiot-proofing [did you really mean to use bitwise here?] and external deprecation...)

Reply:
Same as above. Happy to do a PR for this. Or fix a few of these in a PR.